File: 1768-merge-strings-alternately/1768-merge-strings-alternately.java
class Solution {
    public String mergeAlternately(String word1, String word2) {
       String str = "";
        int len = 0;
        if(word1.length() < word2.length()){
            len = word2.length();
        }else {
            len = word1.length();
        }
        
        for(int i = 0; i < len; i++){
            if(word1.length() - 1 >= i){
                str += word1.charAt(i);            
            }
            if(word2.length() - 1 >= i){
                str += word2.charAt(i);            
            }
        }
        
        System.out.println(str);
        return str;
    }
}

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int len = word1.length() > word2.length() ? word1.length() : word2.length();
        vector<char> arr;
        string ans = "";
        for(int i = 0; i < len; i++){
            if(i < word1.length()){
                arr.push_back(word1[i]);            
            }
            
            if(i < word2.length()){
                arr.push_back(word2[i]);            
            }
        }
        
        for(int i = 0; i < arr.size(); i++){
            ans += arr[i];
        }
        return ans;
    }
};